src/components/pages/Profile/Profile.js

```diff
@@ -1,22 +1,45 @@
-import React, { useEffect, useState } from 'react';
-import { connect } from 'react-redux';
+import React, { useEffect, useState, useMemo } from 'react';
+
+import { useOktaAuth } from '@okta/okta-react';
 
 import RenderGroomerProfile from './RenderGroomerProfile';
 import RenderCustomerProfile from './RenderCustomerProfile';
 
+import { getUserProfileData } from '../../../api';
+
 const Profile = props => {
-  const [isGroomer, setIsGroomer] = useState(null);
+  const { authState, authService } = useOktaAuth();
+  const [memoAuthService] = useMemo(() => [authService], []);
+  const [isGroomer, setIsGroomer] = useState(true);
 
   useEffect(() => {
     // Check current user type to see if groomer or not
+    memoAuthService
+      .getUser()
+      .then(info => {
```

Comment:
Having this in profile.js is much better, good job

Reply:
This was the best approach IMO...good job!